tests/test_exploration.py

```diff
@@ -36,6 +37,9 @@ def test_single_node(self):
         search = BoxSearch(model, parameters, filename="test_single_nodes.hdf")
         search.run()
         search.loadResults()
+        dataarray = search.xr()
+        self.assertTrue(isinstance(dataarray, xr.DataArray))
+        self.assertFalse(dataarray.attrs)
```

Comment:
``` self.assertFalse(dataarray.attrs)```
What does this line do?

Reply:
I am setting attributes of xarray only for MultiModel, so this simply checks whether attributes are empty (empty `dict` evaluates to `False`)